### PR TITLE
Added initial support for discrete movement

### PR DESCRIPTION
**Pull request by EHAT32**
pr corresponding to https://github.com/trueagi-io/minecraft-demo/pull/180

Discrete movement implemented via teleporting agent one block towards given direction.

Also, since I started working with Vereya, I've been encountering issues in commands where agent is supposed to execute commands within the game: i believe the problem is that agent does not have permissions to execute commands when it joins the server. I tried to solve this problem by oping an agent when they join the world by registering the corresponding event. Is it okay to use?

@noskill 
@CICS-Oleg

### CODE REVIEW

File: src/main/java/io/singularitynet/Server/VereyaModServer.java [hunk 1]
```diff
@@ -76,6 +80,18 @@ public void onInitialize() {
...
         });
+
+        ServerPlayConnectionEvents.JOIN.register((handler, sender, server) -> {
+            ServerPlayerEntity player = handler.player;
+            // Grant op status to the player so the commands will work for sure
+            MinecraftServer minecraftServer = player.getServer();
+            if (minecraftServer != null) {
+                GameProfile gameProfile = player.getGameProfile();
+                OperatorEntry operatorEntry = new OperatorEntry(gameProfile, 4, true);
+                minecraftServer.getPlayerManager().getOpList().add(operatorEntry);
```
Comment (noskill) [inline src/main/java/io/singularitynet/Server/VereyaModServer.java]: I think we should limit this operation only for bots, maybe here https://github.com/trueagi-io/Vereya/blob/2cbae3a084b2187bd0abe9fdd2438477bd6278d4/src/main/java/io/singularitynet/Server/ServerStateMachine.java#L773C52-L773C58

Comment (EHAT32) [reply src/main/java/io/singularitynet/Server/VereyaModServer.java]: So I should just put this code in the section you linked to. Is that correct?

Comment (noskill) [reply src/main/java/io/singularitynet/Server/VereyaModServer.java]: yes

**Comment by noskill** [comment]
Test failure is due to your local master branch not being up-to-date with origin/master. Please synchronise your branch.

**Comment by EHAT32** [comment]
Done

**Review by noskill — APPROVED** [review]

**Pull request merged**